### DISCUSSION TITLE
Deleting a scenario in KoboldAI Lite?

### DISCUSSION THREAD

**Discussion by VraethrDalkr** [LostRuins/koboldcpp#138] [Q&A] [answered]
Is it possible to delete a scenario in KoboldAI Lite? I'm using koboldcpp.exe in Windows. I can't figure out where the settings are stored. My guess is that it's using cookies or local storage... or is there a json file somewhere?

**Comment by LostRuins** [comment]
it is hard coded into the HTML file. You can remove it by editing the JSON table near the top of the klite.embd file. 

That said... They're mostly meant as examples - why do you want to delete a scenario?

**Reply by VraethrDalkr** [reply, marked as answer]
Oops! My bad... I thought I've created a new scenario by clicking "New Instruct" then I couldn't figure out how to remove it. Now I understand "New Instruct" is actually a scenario. *facepalm*

**Reply by LostRuins** [reply]
yeap thats the case

**Reply by VL4DST3R** [reply]
Admittedly a bit more control over the scenarios provided (removing but also mainly adding new ones) would be very welcomed. Currently I have to store my templates via the save/load system which has a very limited number of slots (something which also would be great if expanded)

**Reply by LostRuins** [reply]
You can save less frequently used stories to json files on disk. Simply click the "Download" button for that.

**Reply by VL4DST3R** [reply]
I'm already doing that on my end. I'm not saying I haven't found workarounds to make this as painless as possible, just that it could be better (i.e. more save slots in the ui/new db file at least)
